src/v7/hide-graphql-field-suggestions.ts

```diff
@@ -1,5 +1,7 @@
 import { Project, SyntaxKind } from "ts-morph";
 
+export const version = "7.0.0";
```

Comment:
We could also think about treating no version as x.0.0

Reply:
Should we really set the version to all scripts? Then we should make it required. However, I'm against it. For instance, this wouldn't work if upgrading from a beta release to the major version. Scripts added between the beta release and the major wouldn't be executed.